subprojects/apache-orc.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = orc-2.2.0
+source_url = https://www.apache.org/dyn/closer.lua?action=download&filename=orc/orc-2.2.0/orc-2.2.0.tar.gz
```

Comment:
I modeled this off the download link for the Apache Arrow project, but want to double check this is correct @dongjoon-hyun